docs/tutorials/clustering/kclust.ipynb

```diff
@@ -0,0 +1,183 @@
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.cluster import KMeans\n",
+    "from sklearn.metrics import confusion_matrix\n",
+    "from scipy.optimize import linear_sum_assignment\n",
+    "\n",
+    "# Default Sklearn KMeans\n",
+    "k_default_ = KMeans()\n",
+    "\n",
+    "c_hat_k_default = k_default_.fit_predict(X)\n",
+    "\n",
+    "# Function to remap labels for direct comparison\n",
+    "def remap_labels(y_true, y_pred):\n",
+    "\n",
+    "    confusion_mat = confusion_matrix(y_true, y_pred)\n",
+    "    row_inds, col_inds = linear_sum_assignment(confusion_mat, maximize=True)\n",
+    "    label_map = dict(zip(col_inds, row_inds))\n",
+    "    remapped_y_pred = np.vectorize(label_map.get)(y_pred)\n",
+    "    \n",
+    "    return remapped_y_pred\n",
+    "\n",
+    "# Remap Predicted labels\n",
+    "c_hat_kclust = remap_labels(c, c_hat_kclust)\n",
+    "c_hat_k_default = remap_labels(c, c_hat_k_default)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plotting Synthetic clusters\n",
+    "import matplotlib as mp\n",
+    "import seaborn as sns\n",
+    "sns.set()\n",
+    "sns.set_context(\"talk\", font_scale=1.10)\n",
+    "sns.set_style(\"darkgrid\", {\"axes.facecolor\": \".9\"})\n",
+    "\n",
+    "# Labelings\n",
+    "c_list = ['red', 'green', 'blue','orange','purple','yellow','gray']\n",
+    "\n",
+    "# Figure\n",
+    "fig = plt.figure(figsize=(36,9))\n",
+    "\n",
+    "# Synthetic True Clustering\n",
+    "plt.subplot(1, 3, (1,1))\n",
+    "plt.title('True Clustering',fontsize=24,fontweight='bold')\n",
+    "max_c = int(np.max(c))\n",
+    "plt.scatter(X[:,0],X[:,1],c=c,cmap=mp.colors.ListedColormap(c_list[0:max_c+1]))\n",
+    "plt.ylabel('Second Dimension',fontsize=24)\n",
+    "plt.xticks(fontsize=18)\n",
+    "plt.yticks(fontsize=18)\n",
+    "\n",
+    "# Synthetic Kmeans Model\n",
+    "plt.subplot(1, 3, (2,2))\n",
+    "plt.title('Kclust Clustering',fontsize=24,fontweight='bold')\n",
+    "max_c = int(np.max(c_hat_kclust))\n",
+    "plt.scatter(X[:,0],X[:,1],c=c_hat_kclust,cmap=mp.colors.ListedColormap(c_list[0:max_c+1]))\n",
+    "plt.xlabel('First Dimension',fontsize=24)\n",
+    "plt.xticks(fontsize=18)\n",
+    "plt.yticks(fontsize=18)\n",
+    "\n",
+    "# Synthetic Default SKlearn Model\n",
+    "plt.subplot(1, 3, (3,3))\n",
+    "plt.title('Default SKlearn Clustering',fontsize=24,fontweight='bold')\n",
+    "max_c = int(np.max(c_hat_k_default))\n",
+    "plt.scatter(X[:,0],X[:,1],c=c_hat_k_default,cmap=mp.colors.ListedColormap(c_list[0:max_c+1]))\n",
+    "plt.xticks(fontsize=18)\n",
+    "plt.yticks(fontsize=18)\n",
+    "\n",
+    "plt.show()"
```

Comment:
These graphs are pretty small. I liked the size of the autogmm graphs.  Maybe size to match?